.github/workflows/parameters_aws_auth_tests.json.gpg

```diff
@@ -0,0 +1,4 @@
+�
	�.T��Q�Q���_鞋ftS�o��
C���e���ǜ�'�����W����"�#Z8\1j�^?+��h�qGR�,�=���������I�zc8K��=Z��p2��+�kU5�wxƳ
```

Comment:
Maybe it would be better to name it by env name, not the purpose? WDYT?

Reply:
I'd rather leave naming it by purpose, so it is clear right away where it's used